cmd/proxy/main.go

```diff
@@ -23,6 +25,7 @@ var (
 	dockerEnvVars     = flag.String("docker_truststore_env_vars", "", "comma-separated env vars to populate with the proxy cert and patch into containers")
 	dockerJavaEnvVar  = flag.Bool("docker_java_truststore", false, "whether to patch containers with Java proxy cert truststore file and env var")
 	dockerProxySocket = flag.Bool("docker_recursive_proxy", false, "whether to patch containers with a unix domain socket which proxies docker requests from created containers")
+	proxyMode         = flag.String("proxy_mode", "monitor", "mode to run the proxy in. Options: monitor, enforce")
```

Comment:
What's the rough plan for loading the policy at the CLI? do you want to provide a quick-and-dirty json- or file-based flag to load the policy, if provided?

Reply:
yeah that sounds good. want to add a TODO here for the flag?